yellowbrick/utils/nan_warnings.py

```diff
@@ -0,0 +1,31 @@
+"""
+Small helpers that help find missing data.
+"""
+
+import numpy as np
+import warnings
+
+
+def warn_if_nans_exist(data):
+    """Warn if nans exist in a numpy array."""
+    null_count = count_nan_rows(data)
+    total = len(data)
+    percent = null_count / total
+
+    # if np.isnan(data).any():
+    if null_count > 0:
+        warnings.warn(
```

Comment:
If possible, I'd like to create a `YellowbrickWarning` class in `yellowbrick.exceptions`, then we can subclass a `DataWarning` to use here. This will allow us to more easily capture and suppress warnings if needed.

Reply:
This is very similar to how [sklearn does things](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/exceptions.py#L40), so something like:

```python
class YellowbrickWarning(UserWarning):
    """
    Warning class used to notify users of Yellowbrick-specific issues
    """
    pass

class DataWarning(YellowbrickWarning):
    """
    There is an issue with supplied data that may produce unexpected visualizations
    """
    pass
```